src/NzbDrone.Core/Download/TrackedDownloads/DownloadMonitoringService.cs

```diff
@@ -128,12 +144,12 @@ public void Execute(CheckForFinishedDownloadCommand message)
             Refresh();
         }
 
-        public void HandleAsync(EpisodeGrabbedEvent message)
+        public void Handle(EpisodeGrabbedEvent message)
```

Comment:
No coz all it does it trigger the Debouncer, it won't wait for it, less than a msec work.